profiles/views.py

```diff
@@ -97,8 +98,12 @@ class ProfileList(ListCreateAPIView):
 
     permission_classes = [IsAuthenticatedOrReadOnly]
     pagination_class = ForumPagination
-    filter_backends = [django_filters.rest_framework.DjangoFilterBackend]
+    filter_backends = [
+        django_filters.rest_framework.DjangoFilterBackend,
+        filters.OrderingFilter,
+    ]
     filterset_class = ProfileFilter
+    ordering = ["-completeness", "-created_at"]
```

Comment:
This logic with ordering by completeness and created_at in this realization will be applied to all requests to this endpoint.
But if i remember correctly BAs told that we should leave companies list as it is without changes.

Please add ability to send query params from UI client to backend to have an ability to manage what type of sorting, filtering and ordering client wants to have.